vcs/src/main/java/org/openjdk/skara/vcs/git/GitCommits.java

```diff
@@ -135,10 +135,24 @@ public void close() throws IOException {
             }
         }
 
+        RuntimeException runtimeException = null;
+
         for (var i = 0; i < processes.size(); i++) {
             var p = processes.get(i);
             var command = commands.get(i);
-            close(p, command);
+            try {
+                close(p, command);
+            } catch (Exception e) {
+                if (runtimeException == null) {
+                    runtimeException = new RuntimeException(e);
```

Comment:
This converts an IOException to a RuntimeException, which I don't think is ideal. I would suggest just saving the first exception, which should be either a RuntimeException or IOException, and then add any additional exceptions to the suppressed set on the saved exception.